external-docs/docs/guides/configuration/qube-config/columns/units.md

```diff
@@ -0,0 +1,145 @@
+third row uses a different unit, `Centimetres`. In this case, different units are being used to measure the same thing.
+
+## When to use a units column
+
+Every valid data cube needs at least one unit. Units columns can be used in both
+[standard](../../../shape-data/standard-shape.md) and [pivoted shape](../../../shape-data/pivoted-shape.md) cubes.
+
+If every value in an observations column has the same unit, then you should set the unit against the
+[observations column](./observations.md).
+
+If some values in your observations column use one unit and some other values in the same column use a different unit
+then you should use a units column. In the example table above, we see a clear example of this given that the first two
+rows use `Inches` and the third row uses `Centimetres` to measure the `Average Height of Men`.
+
+## Basic configuration
+
+The following JSON shows how a units column can be defined in a [qube configuration file](../index.md):
+
+```json
+{
+    "$schema": "https://purl.org/csv-cubed/qube-config/v1",
+    "title": "Average Height of Men in different countries",
+    "columns": {
+        "Unit": {
+            "type": "units"
+        },
+    }
+}
+```
+
+To define a units column, specify the `type` of the column definition as `units`.
+
+<!-- TODO: At some point, add an example of a single measure pivoted data set with a units column (multiple units) -->
+
+todo: write about default behaviour of units column configuration
+
+## Optional properties
+
+When defining a units column, there are optional properties that can be entered, depending on how your units are
+being defined within the column.
+
+## values
+
+If you are creating new units within your units column, the details of the new units should be entered into a
+`values` field. The JSON below shows an example of the `values` field used in a units column.
+
+```json
+{
+    "$schema": "https://purl.org/csv-cubed/qube-config/v1",
+    "title": "Average Height and Weight for Men in different countries",
+    "columns": {
+        "Unit": {
+            "type": "units",
+            "values": true
+        },
+    }
+}
+```
+
+By default, the `values` field is set to `true`. This indicates to csvcubed to automatically generate
+[unit definitions](../unit-definitions.md) unique to your data set.
+
+## from_template
+
+Units columns can also make use of the `from_template` field. Templates are pre-configured column definitions that help
+speed up the creation of linked data columns. The use of this field in a units column is shown in the example below:
+
+```json
+{
+    "$schema": "https://purl.org/csv-cubed/qube-config/v1",
+    "title": "Average Income in Pounds Sterling",
+    "columns": {
+        "Unit": {
+            "from_template": "http://qudt.org/vocab/unit/PoundSterling"
+        },
+    }
+}
+```
+
+Enter the URI of a template to be used as the units column. Note that you can override the configuration of used
```

Comment:
It's not a URI, it's just an identifier, I think `qudt` is the name of the template which works with units - but you'll need to check that.